public/apps/login/login-page.tsx

```diff
@@ -51,14 +65,15 @@ export function LoginPage(props: LoginPageDeps) {
   const [username, setUsername] = React.useState('');
   const [password, setPassword] = React.useState('');
   const [loginFailed, setloginFailed] = useState(false);
+  const [loginError, setloginError] = useState('');
   const [usernameValidationFailed, setUsernameValidationFailed] = useState(false);
   const [passwordValidationFailed, setPasswordValidationFailed] = useState(false);
 
-  let errorLabel = null;
+  let errorLabel: any = null;
   if (loginFailed) {
     errorLabel = (
       <EuiText id="error" color="danger" textAlign="center">
-        <b>Invalid username or password, please try again</b>
+        <b>{loginError}</b>
```

Comment:
Sorry for re-visiting this topic! This is actually user visible error message. not sure if it really does not need to support i18n. But if translation is not trivial, we could do this in a separate PR if i18n is needed. Just want to make sure that this is indeeded not needed.

Reply:
This is an existing error messages for OSD. If applicable, we can create an issue to validate all the visible error message and apply this changes together.

Reply:
created an issue for follow-up: https://github.com/opensearch-project/security-dashboards-plugin/issues/1161